src/auth/auth_helper.cr

```diff
@@ -0,0 +1,10 @@
+module Google
+  module AuthHelper
```

Comment:
I think this should be `Auth::GetToken` (follows the class/path hierarchy) and the file `get_token.cr` to be more explicit

Reply:
sure